library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterInputTest.java

```diff
@@ -499,7 +499,7 @@ public void testRecoveryOfBlockMetadataIterator() throws InterruptedException
     testMeta.fileSplitterInput.getScanner().setScanIntervalMillis(500);
 
 
-    testMeta.fileSplitterInput.setup(testMeta.context);
```

Comment:
@chandnisingh I already looked at the test during PRs #329 and #333 review. The modified code is clean and readable as well as the original code. The only problem in the original operator code was a wrong time of resource allocation/initialization that is now fixed.